recipes-bsp/firmware-imx/imx-boot-firmware-files_8.18.bb

```diff
@@ -0,0 +1,62 @@
+# Copyright (C) 2018-2020 NXP
+SUMMARY = "Freescale i.MX firmware for 8 family"
+DESCRIPTION = "Freescale i.MX firmware for 8 family"
```

Comment:
not only mx8 and 8m families bu also mx9

Reply:
Fixed in https://github.com/Freescale/meta-freescale/pull/1534